config/clusters/carbonplan/common.values.yaml

```diff
@@ -183,7 +183,6 @@ basehub:
       config:
         Authenticator:
           allowed_users: &users
-            - jhamman
```

Comment:
@maxrjones I'm not right away confident that having an empty list defined here will be okay from a security perspective - that any user may gain access.

I don't have capacity to review that part more in depth right now. Can we add jhamman back for now to avoid needing to look into the details until next week and get this merged now?

Reply:
I've committed Yuvi's suggestion to put in my username. We definitely don't want any user to have access :) I opened a separate ticket to discuss how this works yesterday - so we could continue the discussion there when you have more time. thanks!